styles/FeaturedWork.module.css

```diff
@@ -0,0 +1,84 @@
+.featured-work {
+  margin-top: 2%;
+  margin-bottom: 2%;
+  display: flex;
+  border-bottom: #000000 solid 1px;
+  border-top: #000000 solid 1px;
```

Comment:
Try to use a single border if possible.

Reply:
I tried to figure out how to code border in one line, but failed. I will discuss it in standing-up meeting.

Reply:
Fixed, thanks